schedule-pod-restart/templates/cronjob.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "schedule-pod-restart.name" . }}
+spec:
+  schedule: {{ .Values.cronjob.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: {{ include "schedule-pod-restart.serviceAccountName" . }}
+          containers:
+          - name: kubectl-container
+            image: bitnami/kubectl:{{ include "schedule-pod-restart.bitnamiContainerImageTag" . }}
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - kubectl delete pods -l {{ .Values.cronjob.podLabels }} --namespace={{ .Release.Namespace }}
```

Thinking out loud and not perse for this PR; but can we do this?
```suggestion
            - kubectl get pods -l {{ .Values.cronjob.podLabels }} --namespace={{ .Release.Namespace }}
            - kubectl delete pods -l {{ .Values.cronjob.podLabels }} --namespace={{ .Release.Namespace }}
```

This would list our matching pods first before taking action which would help if pod 2 of 5 fails to understand what was left to delete.

---

It seems that the bitnami container exits after running 1 kubectl statement, so at the moment it's not possible to run multiple kubectl commands (or at least I couldn't make it work this way, maybe there are workarounds for this). We could create a seperate issue for this and add this functionality later on (and leave the single command for now)?

---

No it's ok - This is fine, thank you!